business_requirement/README.rst

```diff
@@ -0,0 +1,285 @@
+
+Followers from the project will automatically be added to the business requirements.
+
+
+Usage
+=====
+
+Simple BR
+^^^^^^^^^
+1. Define the header information
+
+* Master Project (Partner is the one related to the master project)
+* Priority (low, Middle, High)
+* Change request: Is it a change request (currently only informational)
+* Tags: any relevant tag for the business.
+* Owner and approver by default
+
+|image3|
+
+.. |image3| image:: static/img/bus_req_tags2.png
+   :width: 800 px
+   :alt: Input header information
+
+2. Input the customer story, scenario and gap (simple html editor with image and text )
+
+|image4|
+
+.. |image4| image:: static/img/bus_req_cust_story.png
+   :width: 600 px
+   :alt: Input customer story, scenario, gap
+
+3. Confirm the Business requirement (for BR User and Manager)
+   At that stage the customer story/scenario/Gap is not modifiable anymore
+   
+|image5|
+
+.. |image5| image:: static/img/bus_req_confirmed.png
+   :width: 800 px
+   :alt: Confirm your business requirement
+
+4. Approve the Business requirement (for BR Manager)
+
+|image6|
+
+.. |image6| image:: static/img/bus_req_approved.png
+   :width: 800 px
+   :alt: Confirm your business requirement
+   
+5. Once your requirement is finished and delivered you can set it as Done
+
+|image8|
+
+.. |image8| image:: static/img/bus_req_done.png
+   :width: 800 px
+   :alt: Confirm your business requirement
+
+6. Alternatively, you can cancel the BR (in case it is not relevant or mistake) or drop it (when customer makes the decision to discontinue it)
+
+|image9|
+
+.. |image9| image:: static/img/bus_req_cancel.png
+   :width: 800 px
+   :alt: Cancel your business requirement
+
+|image10|
+
+.. |image10| image:: static/img/bus_req_drop.png
+   :width: 800 px
+   :alt: Drop your business requirement
+
+
+Sub-business requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^
+User can create sub business requirements for a given BR for complex cases.
+
+This is a simple parent/child relation (see roadmap).
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/8.0
+
+Known issues / Roadmap
+======================
+
+* Multi-company management
+* Full change request management
+* Analytic account management
+* Complex relationship management
+* Integration with earned-value module.
+* Improve the followers details (eg: depending on the stages)
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/
+project/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
+project/issues/new?body=module:%20
+business_requirement%0Aversion:%20
+8.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

Comment:
This might be the line generating the rst error. Remove the line break.
See http://www.silas.net.br/tech/devel/rst.html